OSDOCS-5442: add info on lvmd and ovn yaml files (#57324)
openshift/openshift-docs: modules/microshift-configuring-ovn.adoc

//
// * microshift_networking/microshift-networking.adoc

:_content-type: PROCEDURE
[id="microshift-config-OVN-K_{context}"]
= OVN-Kubernetes configuration options

{product-title} uses built-in default OVN-Kubernetes values if an OVN-Kubernetes configuration file is not created. You can write an OVN-Kubernetes configuration file to `/etc/microshift/ovn.yaml`.

.Procedure

* To create an `ovn.yaml` file, run the following command:
+
[source, yaml]
----
sudo cat >/etc/microshift/ovn.yaml <<EOF
ovsInit:
  disableOVSInit: false
  gatewayInterface: ""
  externalGatewayInterface: ""
mtu:

> EOF

Author: @ggiguash Hmm, I thought there is no default config file, just built-in values.

> EOF

Reviewer: Giga is right, I forgot that we added those example files to the package. They're examples (note the "default" in the name), and are not actually used by MicroShift. They're there for precisely the reason of these steps in the docs: Copy an existing file with the basic structure you need, then fill in the values you want to change.

----

> .Example 'yaml' configuration file with default values:

Reviewer 2: Here again, I would simply list the file contents
```
$ cat /etc/microshift/ovn.yaml.default
```

An OVN-Kubernetes config file can be written to `/etc/microshift/ovn.yaml`. {product-title} uses default OVN-Kubernetes configuration values if an OVN-Kubernetes config file is not customized.

> //Q: does the file populate with values if created empty, or is none of this true?

Reviewer: I do not see this file on my system, so I don't think anything creates it if the user does not provide one. Generally we would use built-in defaults, but not write them to the filesystem, and I think that's the case here.

> //Q: what's a procedure for creating this file?

Reviewer: There's a default file below. The user just has to create the file using any text editor they prefer. In the docs elsewhere we use the bash here document approach to make it easy to copy and paste, so that would be something like

```
sudo cat >/etc/microshift/ovn.yaml <<EOF
ovsInit:
  disableOVSInit: false
  gatewayInterface: ""
  externalGatewayInterface: ""
mtu: 1400
EOF
```

1. The OVS bridge is required. When `disableOVSInit` is true, OVS bridge `br-ex` must be configured manually.
--

[IMPORTANT]
====

> If you change the `mtu` configuration value in the `ovn.yaml` file, you must restart the host that {product-title} is running on for the updated setting to apply.

Reviewer 3: ```suggestion
If you change the `mtu` configuration value in the `ovn.yaml` file, you must restart the host that {product-title} is running on to apply the updating setting.
```

====

> .Example custom `ovn.yaml` configuration file:

Reviewer 3: ```suggestion
.Example custom `ovn.yaml` configuration file
```

====

> When `disableOVSInit` is set to true in the `ovn.yaml` config file, the OVS bridge br-ex must be manually configured.

Reviewer 3: ```suggestion
When `disableOVSInit` is set to true in the `ovn.yaml` config file, the OVS bridge `br-ex` must be manually configured.
```

====

> 

Reviewer 3: Should this note be indented since it's applying to an item in the table? I understand why it's not a footnote so it doesn't get lost but maybe the indent would make it a bit more clear that it applies to an item in the table.
